flocker/provision/_digitalocean.py

```diff
@@ -206,6 +209,52 @@ def latest_droplet_kernel(droplet,
     return latest_kernel
 
 
+def change_kernel(node_id, token, kernel_version):
+    """
+    Change the configured kernel on a digitialocean node.
+
+    :param bytes node_id: The id of the digitialocean.
+    :param bytes token: A DigitalOcean v2 API token.
+    """
+    # Import here, so that this can be added to ``flocker.provision`` without
+    # having to install ``pyocean``.
+    import pyocean
```

Comment:
:question: This is an existing comment but I don't understand why this needs to be the case - where is being run that we can't / don't want to install dependencies?